src/PublishEvent.php

```diff
@@ -13,9 +15,7 @@ class PublishEvent implements JsonSerializable {
 	public array $attributes;
 
 	public function jsonSerialize(): object {
-		$object = (object) [
-			'publishedAt' => $this->publishedAt,
-		];
+		$object = new stdClass();
 
 		foreach (['hashed', 'exposures', 'attributes'] as $key) {
```

Comment:
It seems like exposures are already taken care of in the loop with the goals below (only visible here on github if you expand the code below), right?

Reply:
You are right that was mixed up there. Fix coming up.